src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,37 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../helpers/Todo';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todo,
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    id,
+    title,
+    user,
+    userId,
+    completed = false,
+  } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={classNames('TodoInfo', {
+        'TodoInfo--completed': completed,
+      })}
+    >
+      <h2 className="TodoInfo__title">
+        {title}
+      </h2>
+
+      {
+        user && (
+          <UserInfo user={user} key={userId} />
```

Comment:
why do we need to pass key here?
we usually pass key when looping the array, so in this case, you can remove this prop